src/platform/packages/shared/kbn-alerts-as-data-utils/src/schemas/generated/security_attackdiscovery_schema.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+// ---------------------------------- WARNING ----------------------------------
+// this file was generated, and should not be edited by hand
```

Comment:
Usually these files are generated from files with ES mappings, I believe mainly in the peer directory `src/field_maps`. Not seeing where it might have been generated for.

Which makes me wonder if we've defined mappings for these fields in the alerts documents.  Perhaps this is done differently for security rules, or this is part of a feature branch where some of that work was already done.  

In any case, we should make sure we have the ES mappings in place for these field, if we want them indexed.

Reply:
The mappings are defined in this file https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/elastic_assistant/server/lib/attack_discovery/schedules/fields/field_map.ts.

I followed other solutions including security while adding these mappings and this new alert type. Similarly to o11y, ml, security, stack etc. the mappings are defined inside those solutions and the generated schemas then added via FTR test script inside the `src/platform/packages/shared/kbn-alerts-as-data-utils/src/schemas/generated/`.

Here is example of similarly defined mappings https://github.com/elastic/kibana/blob/main/x-pack/solutions/observability/plugins/apm/server/routes/alerts/register_apm_rule_types.ts